lib/components/Footer/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { Text } from '../Text'
+import classNames from 'classnames'
+import { FooterProps } from './types'
+import { certificatesMap, socialsMap } from './data'
+import './styles.scss'
+
+export const Footer = ({
+  logo,
+  categoryLinks,
+  socialLinks,
+  address,
+  certificates,
+  notes,
+  copyrights,
+}: FooterProps) => {
+  const usedCertificates = certificates.map(
+    (certificate) => certificatesMap[certificate],
+  )
+
+  const usedSocials = Object.keys(socialLinks).map((social) => {
+    return {
+      ...socialsMap[social as keyof typeof socialsMap],
+      url: socialLinks[social as keyof typeof socialLinks],
+    }
+  })
+
+  const handleClick = (url: string) => {
+    window.open(url, '_blank')
+  }
+
+  return (
+    <footer className="au-footer">
+      <section className="au-footer__logo">{logo}</section>
+      <section className="au-footer__links">
+        {categoryLinks.map(({ categoryTitle, links }, index) => {
+          return (
+            <div key={index} className="au-footer__links__category">
+              <Text as="h2" variant="heading-micro" weight="bold">
+                {categoryTitle}
+              </Text>
+              {links.map(({ title, url }, index) => {
+                return (
+                  <div
+                    className={classNames('au-footer__links', {
+                      clickable: !!url,
+                    })}
+                    key={index}
+                    onClick={() => url && handleClick(url)}>
+                    <Text
+                      as="a"
+                      variant="body-medium"
+                      weight="regular"
+                      color="secondary">
+                      {title}
+                    </Text>
+                  </div>
+                )
+              })}
+            </div>
+          )
+        })}
+        <div className="au-footer__links__category">
+          <Text as="h2" variant="heading-micro" weight="bold">
+            Siga a gente
+          </Text>
+          <div className="au-footer__links__socials">
+            {usedSocials.map(({ icon, url }, index) => {
+              return (
+                <div
+                  className={classNames('au-footer__links', {
+                    clickable: !!url,
```

Comment:
aqui sempre nas classes que são modificadoras condicionalmente no BEM o padrão é usar o `--` ou prefixo `is-`

então ficaria
`au-footer__links--clickable` ou `is-clickable` versão short

Referência na doc do Aurora
https://aurora.consumidorpositivo.com.br/?path=/docs/padr%C3%B5es-de-c%C3%B3digo--docs#bem-css